internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -273,6 +288,44 @@ func (opts *Opts) askDBUserAccessListOptions() error {
 	return nil
 }
 
+func (opts *Opts) askMongoShellQuestion() (bool, error) {
+	if opts.SkipMongoShellQuestions {
+		return false, nil
+	}
+
+	if opts.ConfigMongoShellPath() == "" {
+		answer := false
```

Comment:
[nit] can we give `answer` a more meaningful name? I can tell is the answer to `newMongoShellQuestionNotFound` but what does it mean it being `false` or `true`? it would make easier to read `if answer {` as right now I have no idea what that line means